tests/nodeos_run_test.py

```diff
@@ -549,7 +549,7 @@
     action="transfer"
     data="{\"from\":\"defproducera\",\"to\":\"currency1111\",\"quantity\":"
     data +="\"00.0025 CUR\",\"memo\":\"test\"}"
-    opts="--permission defproducera@active"
+    opts="--permission defproducera@active --use-old-send-rpc"
```

Comment:
Why do these need to use the old rpc? Can that be updated?

Reply:
They could now be updated but I thought it would be useful to include at least a couple of tests that still use the other RPC API, so it doesn't stop being tested.  As long as it exists, we should continue verifying its functionality.

Reply:
Can you add a comment to that affect, so it is obvious this is here to test that and can be easily removed in the future.

Reply:
Comment added.